web/styles/informational_overlays.css

```diff
@@ -65,6 +65,10 @@
         /* keeps dividing line at same width for all tables in model. */
         width: calc(50% - 11px);
         text-align: right;
+
+        & img.emoji {
+            margin-bottom: 7px;
```

Comment:
Let's not do this; study how inline emoji are presented in all rendered markdown areas. The "React to selected message with 👍🏻" needs to take the same stack of CSS variables, and then it should be perfectly aligned with the surrounding text.

Reply:
Thanks for the valuable feedback.
On studying the inline emoji I found that the above problem can indeed be solved using the following modification:
```
& .emoji {
            margin: var(--length-line-oversize-block-margin-adjust) auto;
            vertical-align: sub;
        }
```

### Screenshots

| Before Changes | After Changes |
|----------|----------|
| ![image](https://github.com/user-attachments/assets/a0c23180-9a63-4620-8cfb-30800e95776d)| ![image](https://github.com/user-attachments/assets/c601e915-2cef-451c-a197-e43b5a5a3a60)|

I would like to hear what you think about this before moving forward.

Reply:
The styles here should match the `:heart:` emoji on the Message formatting pane, specifically:

![Screenshot 2024-10-14 at 2 47 41 PM](https://github.com/user-attachments/assets/24af399f-8ed7-4473-87e4-1bab51d09a64)

There's no need to invent hacky workarounds like `vertical-align: sub`

Reply:
I tried the approach mentioned above and got the following result:

<details> 
<summary> Screenshot of thumbs-up emoji</summary>

![image](https://github.com/user-attachments/assets/766c1cd9-463a-4bb7-8761-498dd7bb81e2)

</details>

On examining the emoji, I think it is due to the shape of the thumbs-up emoji specifically that it looks like it is misaligned if we replace the thumbs-up emoji with a heart emoji, it aligns well. (Screenshot follows)

<details> 
<summary> Screenshot of heart emoji</summary>

![image](https://github.com/user-attachments/assets/22403853-a3df-467e-b3fc-cd631ff3dede)

</details>

That's why I figured to use the `vertical-align: sub` property.